digiroad2-oracle/src/main/resources/db/migration/V236__create_LengthOfRoadAxis_properties.sql

```diff
@@ -0,0 +1,116 @@
+-- NEW ASSET TYPE
+INSERT INTO ASSET_TYPE(ID, NAME, GEOMETRY_TYPE, CREATED_BY )
+    VALUES (450,'Pituussuuntaiset tiemerkinnät','linear','db_migration_v236');
+
+-- LOCALIZED_STRING VALUE
```

Comment:
This code only will be executed once per environment, maybe these validations are not useful at all.
We can perform a fast manual validation to confirm if the LOCALIZED_STRING already exists instead of trying to insert all again each time.